storage/shard.go

```diff
@@ -1075,6 +1075,7 @@ func (s *dbShard) FetchBlocks(
 	ctx context.Context,
 	id ident.ID,
 	starts []time.Time,
+	blockDuration time.Duration,
```

Comment:
don't need to take this as an arg, the shard has it available via: `dbShard.namespace.Options().RetentionOptions().BlockSize()`